Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -4738,29 +4765,23 @@ public void StopDynamicBrakeIncrease()
 
         public void StartDynamicBrakeDecrease(float? target)
         {
-            if (Direction == Direction.N)
+            // Only allow decreasing dynamic brake if dynamic brake isn't off
+            // Note: For safety, this will allow decreasing the dynamic brake even if a mechanical interlock should have locked dynamic brake in off position
+            if (DynamicBrakeController.CurrentValue > 0)
             {
-                Simulator.Confirmer.Warning(CabControl.DynamicBrake, CabSetting.Warn1);
-                return;
-            }
-            AlerterReset(TCSEvent.DynamicBrakeChanged);
-            if (!CanUseDynamicBrake())
-                return;
+                DynamicBrakeController.StartDecrease(target);
```

Comment:
Getting a crash if I press the "," key in HHP-8 (no dynamic brake controller

Reply:
Didn't even consider testing that. Fixed in next build.